ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -425,6 +425,149 @@ presubmits:
+    rerun_command: /test e2e-aws-multitenant
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-multitenant-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-multitenant
+        - --template=/usr/local/e2e-aws-multitenant
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_NETWORK_MANIFEST
+          value: |
+            apiVersion: operator.openshift.io/v1
+            kind: Network
+            metadata:
+              name: cluster
+            spec:
+              defaultNetwork:
+                type: OpenShiftSDN
+                openshiftSDNConfig:
+                  mode: Multitenant
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-origin-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-aws-multitenant
+        - name: TEST_COMMAND
+          value: TEST_SUITE=openshift/conformance/parallel run-tests
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-aws-multitenant-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-aws-multitenant
+          name: job-definition
+          subPath: cluster-launch-installer-e2e.yaml
+        - mountPath: /etc/sentry-dsn
+          name: sentry-dsn
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-aws
+      - configMap:
+          name: prow-job-cluster-launch-installer-e2e
+        name: job-definition
+      - name: sentry-dsn
+        secret:
+          secretName: sentry-dsn
+    trigger: (?m)^/test( | .* )e2e-aws-multitenant,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-ovn-kubernetes
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-origin-master-e2e-aws-ovn-kubernetes
+    optional: true
+    rerun_command: /test e2e-aws-ovn-kubernetes
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-ovn-kubernetes-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-ovn-kubernetes
+        - --template=/usr/local/e2e-aws-ovn-kubernetes
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_NETWORK_TYPE
```

Comment:
I would also expect a periodic release job (if you don’t have one already)

Reply:
I think we were waiting until we got things working for that...

Reply:
(added https://jira.coreos.com/browse/SDN-580 so we don't forget to do this)